src/CLUBB_core/advance_helper_module.F90

```diff
@@ -683,10 +698,25 @@ subroutine compute_Cx_fnc_Richardson( nz, ngrdcol, gr, &
     ! Cx_fnc_Richardson is interpolated based on the value of Richardson_num
     ! The min function ensures that Cx does not exceed Cx_max, regardless of the
     !     value of Richardson_num_max.
-    Cx_fnc_Richardson = linear_interp_factor( &
-                    ( max(min(Richardson_num_max,Ri_zm),Richardson_num_min) &
-                    - Richardson_num_min )  * invrs_min_max_diff, &
-                                              clubb_params(iCx_max), clubb_params(iCx_min) )
+    if (l_modify_limiters_for_cnvg_test) then 
```

Comment:
@vlarson: Yes. The only reason I added a flag "l_modify_limiters_for_cnvg_test" here is: I noticed that l_smooth_min_max is added to many places, and we did not test all of them in our convergence paper. Therefore, I think it may be better for me to clearly marked it to avoid misleading.  However, I think you may already test the l_smooth_min_max as it has been implemented in the CLUBB master branch. Therefore, it would make sense to use l_smooth_min_max.